packages/pn-pa-webapp/src/api/external-registries/external-registries-routes.ts

```diff
@@ -7,21 +7,45 @@ const API_EXTERNAL_REGISTRIES_PREFIX = 'ext-registry';
 const PA_SEGMENT = 'pa';
 const API_VERSION_SEGMENT = 'v1';
```

Comment:
we are not using version segment anymore. check notification.routes to see how do it

Reply:
Duplicate